### PR TITLE
Updating troubleshooting ZE content for Zowe V3 - first draft

### PR DESCRIPTION
**Pull request by anaxceron**
Describe your pull request here: Updating troubleshooting ZE content for Zowe V3.

List the file(s) included in this PR: ze-issues.md +

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

File: docs/troubleshoot/cli/cli-issue.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Raising a CLI issue on GitHub
 
-When necessary, you can raise GitHub issues against the Zowe&trade; CLI repository [here](https://github.com/zowe/zowe-cli/issues). It is suggested that you use either the bug or enhancement template.
+When necessary, raise a GitHub issues in the [Zowe&trade; CLI repository](https://github.com/zowe/zowe-cli/issues). It is suggested that you use either the bug or enhancement template.
```
Comment (traeok) [inline docs/troubleshoot/cli/cli-issue.md:3]: Plural form `issues` -> `issue`:
```suggestion
When necessary, raise a GitHub issue in the [Zowe&trade; CLI repository](https://github.com/zowe/zowe-cli/issues). It is suggested that you use either the bug or enhancement template.
```

File: docs/troubleshoot/ze/ze-issues.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Raising a Zowe Explorer issue on GitHub
 
-You can raise GitHub issues against [the Zowe Explorer repository](https://github.com/zowe/zowe-explorer-vscode/issues). It is suggested that you use either the bug or feature request.
+When necessary, raise a GitHub issues in the [Zowe Explorer for Visual Studio Code repository](https://github.com/zowe/zowe-explorer-vscode/issues). It is suggested that you use either the bug or feature request.
```
Comment (traeok) [inline docs/troubleshoot/ze/ze-issues.md:3]: Same suggestion for `issues -> issue`:

```suggestion
When necessary, raise a GitHub issue in the [Zowe Explorer for Visual Studio Code repository](https://github.com/zowe/zowe-explorer-vscode/issues). It is suggested that you use either the bug or feature request.
```

File: docs/troubleshoot/ze/ze-known-limits.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ With Remote SSH, the network protocol provides users with a secure connection to
 
 Both WSL and Remote SSH provide more tools for mainframe developers, but they also have limitations when it comes to credentials.
 
-Authentication information used in Zowe Explorer &mdash; such as usernames and passwords, SSH keys, API Mediation Layer tokens &mdash; resides on the developer’s local machine, not the Linux environment or the remote server or virtual machine. This is because credential storage is managed by VS Code, which stores them using the host's operating system credential manager.
+Authentication information used in Zowe Explorer &mdash; such as usernames and passwords, SSH keys, API Mediation Layer tokens &mdash; resides on the developer’s local machine, provided the developer is not connected to a remote environment through VS Code. If you are connected to a remote environment through VS Code, your secure credentials are stored on the remote system.
```
Comment (traeok) [inline docs/troubleshoot/ze/ze-known-limits.md:15]:
```suggestion
Authentication information used in Zowe Explorer &mdash; such as usernames and passwords, SSH keys, and API Mediation Layer tokens &mdash; reside on the developer’s local machine, provided the developer is not connected to a remote environment through VS Code. If you are connected to a remote environment through VS Code, your secure credentials are stored on the remote system.
```

**Review by traeok — COMMENTED** [review]
LGTM! Left a couple comments about phrasing, but content looks good 👍 
Thanks Ana

File: docs/troubleshoot/ze/ze-known-limits.md (hunk 3 above)
Comment (anaxceron) [reply docs/troubleshoot/ze/ze-known-limits.md:15]: @traeok -- technically, "information" is singular, therefore "resides" is correct.

**Review by adam-wolfe — APPROVED** [review]
LGTM, thanks @anaxceron

**Pull request merged**